recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "pythonanywhere" %}
-{% set version = "0.11.0" %}
+{% set version = "0.15.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 build:
   number: 0
-  skip: True  # [py<37]
+  skip: True  # [py<38]
   script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 27d0111acb70ae751bf117a451d4a4826ae3fe1ac221b0e1c3a0044bf7621172
+  sha256: 6abab25d405111a2a0ab94f05d06dc0e2958bc53af082224dd3012f86b1840d9
 
 requirements:
```

Comment:
`pythonanywhere_core==0.2.4` is required, see https://github.com/pythonanywhere/helper_scripts/blob/v0.15.5/setup.py#L41C10-L41C36. It was added in v0.15.3 https://github.com/pythonanywhere/helper_scripts/commit/b5ea2293091291a95c1e0ebadca6b0dd7076724f

You should create a brand new `pythonanywhere_core-feedstock `first. See the docs on how to do it https://github.com/anaconda/perseverance-skills/blob/main/sections/02_Package_building/01_How_tos/Creating_new_feedstocks.md. As there is no conda-forge feedstock, your main tool for generating the recipe will be `grayskull` https://github.com/conda/grayskull